content/ways-to-train.md

```diff
@@ -1,20 +1,17 @@
 ---
-title: "Ways to train"
+title: "How to train"
```

Comment:
What's the rationale for this change?

Reply:
to make it feel like you have fewer decisions to make - make it more straightforward - to make it more active